src/app/components/details/album_header.css

```diff
@@ -1,14 +1,21 @@
 .title-4 label {
```

Comment:
is title-4 a standard class? I think it is! if that's the case, I'd rather have us define an additional class for the artist link, so that we don't have affect other possible uses of title-4 in the future